src/server-manager.cpp

```diff
@@ -100,7 +100,7 @@ constexpr const uint64_t ROLLUP_ADVANCE_STATE = 0;
 constexpr const uint64_t ROLLUP_INSPECT_STATE = 1;
 
 static constexpr uint32_t manager_version_major = 0;
-static constexpr uint32_t manager_version_minor = 7;
+static constexpr uint32_t manager_version_minor = 8;
```

Comment:
Should we open an issue to handle this in the Makefile, the same as we do in `machine-emulator`?

Reply:
Yes please

Reply:
Created an issue for it: https://github.com/cartesi/server-manager/issues/10